jenkins-builds/tools/mf.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+usage() {                                                                    
+    echo "Usage: $0 <daint-gpu|daint-mc>"
+    exit 1;
+}
+
+partition=$1
+if [ -z $partition ] ;then
+    usage
+fi
+
+### jenkinslist=$productiondir/jenkins-builds/6.0.UP04-17.08-gpu
+### jenkinslist=$productiondir/jenkins-builds/daint-jg
+#jenkinslist=$productiondir/jenkins-builds/6.0.UP04-17.08-mc
+#jenkinslist=daint-jg
+productiondir=/apps/common/UES/sandbox/jgp/production.git
```

Comment:
why do you point to a specific git repository? if this script is being added to the production repository, one would expect that it should use the files on the same repo